packages/terra-clinical-onset-picker/docs/README.md

```diff
@@ -0,0 +1,28 @@
+# Terra Clinical Onset Picker
+
+The terra-clinical-onset-picker component provides users a way to enter or select an approximate date for use in onset scenarios.
+
+This is a controlled component because it manages the state of the value in the input. Because this is a controlled input component, it cannot accept the defaultValue prop as it always uses the value prop. React does not allow having both the defaultValue and value props.
```

Comment:
Is onset-picker always a controlled component? Consumers might not know what the defaultValue prop means if its not a first class prop they must choose to use.

Reply:
I agree. I wouldn't put implementation details in the readme. Stating the the component manages it's own state should be sufficient.

Reply:
I think I was repeating wording used on other components, but yeah not that useful. Slimmed it down to just "This component manages its own state."